creator-node/src/ImageProcessingQueue.ts

```diff
@@ -45,7 +45,17 @@ export class ImageProcessingQueue {
     })
 
     // Process jobs sandboxed - https://docs.bullmq.io/guide/workers/sandboxed-processors
-    const processorFile = path.join(__dirname, 'resizeImage.js')
+    let processorFile
+
+    // run the sandbox worker from the transpiled js resizeImage file
+    // this cannot import typescript so we need to give it vanilla js
+    // during local dev we're in src/, but prod starts in build/src
+    // see scripts/start.sh for entry points
+    if (__dirname.includes('/build/src')) {
```

Comment:
Might be useful to have a `buildDir` env var or something. Keeping info like this in source code is gonna bite us in the ass if we keep it there for too long